ppr-ui/src/resources/attnRefConfigs.ts

```diff
@@ -1,16 +1,24 @@
 import { AttnRefConfigIF } from '@/interfaces'
 
-export const clientConfig: AttnRefConfigIF = {
+export const folioOrRefConfig: AttnRefConfigIF = {
   title: 'Folio or Reference Number',
   description: 'Add an optional number for this transaction for your own tracking purposes. This information is not' +
     ' used by the Manufactured Home Registry.',
   inputTitle: 'Folio Number',
   inputLabel: 'Folio or Reference Number (Optional)'
 }
 
-export const staffConfig: AttnRefConfigIF = {
+export const attentionConfig: AttnRefConfigIF = {
   title: 'Attention',
   description: 'If the intended recipient of the registry documents is different from the Submitting Party' +
     ' name, add an optional name. If entered, it will appear on the cover letter.',
+  inputTitle: 'Attention',
   inputLabel: 'Attention (Optional)'
 }
+
+export const attentionConfigManufacturer: AttnRefConfigIF = {
+  ...attentionConfig,
+  description: 'If the intended recipient of the registration verification statement and decal is' +
```

Comment:
Here I would suggest keeping the static content sting intact. It will make it easier to search and compare for any typos/errors with the source. You can use `// eslint-disable-line max-len` for it.

Reply:
I find it harder to read, when I have to scroll quite a bit. I haven't issues with checking for typos yet.

Reply:
Yea, I know we had them. I'm trying to change that and have the text in one line. So in case the text gets updated, I don't have to check each line and just paste it as is. Im ok leaving it as is if still not convinced 😄